GearSpawner/Patches.cs

```diff
@@ -1,34 +1,46 @@
 ﻿using HarmonyLib;
 using Il2Cpp;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
 
 namespace GearSpawner;
 
 internal static class Patches
 {
-	[HarmonyPatch(typeof(LootTable), nameof(LootTable.GetPrefab))]
-	internal static class LootTable_GetPrefab
+	// patch the look tables as they initialize
+	[HarmonyPatch(typeof(LootTableData), nameof(LootTableData.Initialize))]
+	internal static class LootTableData_Awake
 	{
-		private static void Prefix(LootTable __instance)
+		private static void Postfix(LootTableData __instance)
 		{
-			LootTableManager.ConfigureLootTable(__instance);
+				bool changed = LootTableManager.ConfigureLootTableData(__instance);
 		}
 	}
 
-	[HarmonyPatch(typeof(LootTable), nameof(LootTable.GetRandomGearPrefab))]
-	internal static class LootTable_GetRandomGearPrefab
+	//patch the scenes for loose items as they load
+	[HarmonyPatch(typeof(QualitySettingsManager), nameof(QualitySettingsManager.ApplyCurrentQualitySettings))]//Exists
```

Comment:
I want a documentation comment about why we patch this particular method.

Other than `GameManager.SetAudioModeForLoadedScene()`, `QualitySettingsManager.ApplyCurrentQualitySettings` is the last method called within `GameManager.Update()` before save file saving and loading occur. They only get called after the loading panel has closed, and they each only get called once. If `GameManager.SetAudioModeForLoadedScene()` was not inlined, it would be used instead.